modules/network/locals.tf

```diff
@@ -238,19 +246,35 @@ locals {
       port             = -1,
       stateless        = false
     },
+    {
+      description      = "Pod to Kubernetes API endpoint communication.",
+      destination      = local.cp_subnet,
+      destination_type = "CIDR_BLOCK",
+      protocol         = local.tcp_protocol,
+      port             = 6443,
+      stateless        = false
+    },
+    {
+      description      = "Pod to Kubernetes API endpoint communication.",
+      destination      = local.cp_subnet,
+      destination_type = "CIDR_BLOCK",
+      protocol         = local.tcp_protocol,
+      port             = 12250,
+      stateless        = false
+    }
   ]
 
   pods_ingress = [
     {
-      description = "Allow worker nodes to access pods.",
+      description = "Kubernetes API endpoint to pod communication.",
       protocol    = local.all_protocols,
       port        = -1,
       source      = local.cp_subnet,
       source_type = "CIDR_BLOCK",
       stateless   = false
     },
     {
-      description = "Allow Kubernetes Control Plane to communicate with pods.",
+      description = "Allow pods on one worker node to communicate with pods on other worker nodes.",
       protocol    = local.all_protocols,
       port        = -1,
       source      = local.workers_subnet,
```

Comment:
Should the source be the pod subnet? 

Also, can we be more specific re: protocols and ports?

Reply:
Updating the description to be more clear; this is the other side of the `workers_ingress_npn` change further below, to support cross-host `NodePort -> Pod` or `hostNetwork pod -> pod`.